src/SingleCellAnnotator.imjoy.html

```diff
@@ -473,6 +479,15 @@
       });
       api.alert(JSON.stringify(blob))
     },
+    // go to nextImage after select yes/no.
+    async singleSelected(){
```

Comment:
Perhaps a better name, say, `selectLocation(true/false)` with a parameter for selected or not so we don't need two functions for this.

Reply:
Also there is another place with the Y/N keybinding you need to change.

Reply:
added